source/darwin/secure_transport_tls_channel_handler.c

```diff
@@ -1056,6 +1056,12 @@ struct aws_tls_ctx *aws_tls_client_ctx_new(struct aws_allocator *alloc, const st
 }
 
 void aws_tls_ctx_destroy(struct aws_tls_ctx *ctx) {
+
+    if (ctx == NULL) {
+        AWS_LOGF_ERROR(AWS_LS_IO_TLS, "static: trying to destroy a NULL TLS Context.");
```

Comment:
This is a warning at worst, but since we're safely able to bail and maintain idempotency, I don't think this is necessary at more than TRACE.